### ISSUE TITLE
Bug in rolling attack/damages on homebrew monster

### ISSUE THREAD

**Issue by kakaroto** [kakaroto/Beyond20#426]
https://www.dndbeyond.com/characters/18668666/oQgFhD
Extra -> Beast Companion -> Cricket
![image](https://user-images.githubusercontent.com/27990/86527491-21712f80-be6d-11ea-84f1-6480f8e34f24.png)
It just rolls 0, but it shows the right formula, and there's no damages in the message B20 sends.

**Comment by HaarisQureshi** [comment]
Thank you!

**Comment by kakaroto** [comment]
Someone in the chrome store reported a similar issue but different way to trigger it : 
> 1. When using adv/dis when rolling from ddb, the result always shows as 0. If I click on the result in the chat in foundry it shows the correct dice results. I’m guessing this might be because this is a new feature on ddb and will be fixed in an upcoming version, but this also happens when... 
> 2. I make the same ability check two times in a row (ex 2 perception checks in quick succession). The first one displays correctly, the second one always shows a 0. Any idea why this happens?

**Comment by kakaroto** [comment]
Two issues found : 
1 - Your hit values were not detected because you didn't have a space between the "Hit:" and the values... yes, it looks like a space, but it was actually the html element `&nbsp;` (non-breaking-space), which is why it wasn't detected. I've fixed it so it unescapes special characters like that.
2 - There was another bug where it wasn't actually rolling/resolving the to-hit rolls if you had no damages.

Thanks for reporting it and the link to a reproducible sheet. Both issues are now fixed.
